src/System.Console/tests/System.Console.Tests.builds

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <ItemGroup>
-    <Project Include="System.Console.Tests.csproj" />
```

Comment:
I guess we don't really need to remove them yet, so in order to be able to merge without breaking @chcosta then we can just undo this delete.